dawn/src/dawn/Serialization/ASTSerializer.cpp

```diff
@@ -738,6 +739,8 @@ makeFieldDimensions(const proto::statements::FieldDimensions& protoFieldDimensio
                       "serialized FieldDimensions message.");
 
       NeighborChain neighborChain;
+      neighborChain.push_back(
+          getLocationTypeFromProtoLocationType(protoUnstructuredDimension.dense_location_type()));
```

Comment:
actually "sparse part" is already a neighbor chain (it doesn't need to be completed with the dense location, because it is already there) ... or at least that's how I thought it, but we can discuss